trino/sqlalchemy/dialect.py

```diff
@@ -216,8 +217,9 @@ def get_view_names(self, connection: Connection, schema: str = None, **kw) -> Li
         query = dedent(
             """
             SELECT "table_name"
-            FROM "information_schema"."views"
+            FROM "information_schema"."tables"
             WHERE "table_schema" = :schema
+              AND "table_type" = 'VIEW'
```

Comment:
Please add a test covering this (probably in `test_sqlalchemy_integration`).

You can create a VIEW in the memory connector and verify that it's not present in output of `get_table_names` and is present in `get_view_names`.

Reply:
@hashhar are you aware that there's no current unit/integration tests for either the `get_table_names` and `get_view_names` methods?

Reply:
yes I do (unfortunately). SQLAlchemy integration made it in because it was useful but there hadn't been much time invested into it (until very recently).